build.gradle

```diff
@@ -30,22 +30,25 @@ repositories {
 }
 
 dependencies {
-    testImplementation 'commons-io:commons-io:2.+'
-    testImplementation 'org.mockito:mockito-core:4.+'
-    testImplementation 'org.assertj:assertj-core:3.+'
-    testImplementation 'org.hamcrest:hamcrest-core:2.+'
-    testImplementation 'org.apache.commons:commons-lang3:3.+'
-    testImplementation 'com.h2database:h2:2.+'
-    
+    testImplementation 'commons-io:commons-io:2.11.0'
+    testImplementation 'junit:junit:4.13.2'
+    testImplementation 'org.mockito:mockito-core:4.5.1'
+    testImplementation 'org.assertj:assertj-core:3.22.0'
```

Comment:
Is there a reason to explicitly list these here (compared to the 'older' on the left, where only the major version is fixed?